### DISCUSSION TITLE
Duda Parcial 1 Parte 2

### DISCUSSION THREAD

**Discussion by Bienvee3** [mmasias/24-25-PRG1#582] [General]
Tenia una duda, es que si cada vez que los niños salen de un edificio vacían la bolsa? Porque al ejecutar el código ya completo, solo suelen llegar al edificio 2 y como tienen todos las bolsas llenas no visitan más los demás edificios y se termina. ¿Asi estaría bien?

**Comment by mmasias** [comment]
Dale un vistazo a https://github.com/mmasias/24-25-PRG1/discussions/581#discussioncomment-11157263
